src/nemo-desktop-window.h

```diff
@@ -47,7 +47,6 @@ typedef struct NemoDesktopWindowDetails NemoDesktopWindowDetails;
 typedef struct {
 	NemoWindow parent_spot;
 	NemoDesktopWindowDetails *details;
-        gboolean affect_desktop_on_next_location_change;
```

Comment:
affect_desktop_on_next_location_change is not used. So we can keep this unrelated change.